Enhance Print Grammar to print whole grammar or subsets of it. (#16440)
coq/coq: doc/sphinx/user-extensions/syntax-extensions.rst

   Prints the current reserved :ref:`keywords <keywords>` and parser tokens, one
   per line. Keywords cannot be used as identifiers.

.. cmd:: Print Grammar {* @ident }

   When no :token:`ident` is provided, shows the whole grammar.

>    Otherwise shows the grammar for the nonterminal :token:`ident`\s. The
>    following are treated specially and print some related grammars:

Reviewer: ```suggestion
   Otherwise shows the grammar for the nonterminal :token:`ident`\s, except for
   the following, which will include some related nonterminals:
```
